ruby/hamsters.rb

```diff
@@ -0,0 +1,35 @@
+puts "What is the hamster's name?"
+
+name = gets.chomp
+
+puts "Volume level (1-10)"
+
+volume = gets.chomp.to_i
+
+if volume == 0
+  volume = nil
+else
+end
+
+puts "Fur color"
+
+furcolor = gets.chomp
+
+puts "Good candidate for adoption? (y/n)"
+
+candidate = gets.chomp
+
+puts "Estimated age?"
+
+age = gets.chomp.to_i
+
+if age == 0
+  age = nil
+else
+end
+
+puts "Hamster Name: #{name}"
+puts "Volume Level: #{volume}"
+puts "Fur Color: #{furcolor}"
+puts "Candidate For Adoption: #{candidate}"
+puts "Hamster Age: #{age}"
```

Comment:
consider p vs puts when working with explicit returns.

puts is an explicit return that will print out the assigned arg to a string in the console. puts also returns nil

p on the other hand with put out the value of the code to the right of is as well as return the object. This may not make sense now but using puts as your return statement can cause some major trouble in the future. Please be wary